x-pack/test/functional/services/ml/common_ui.ts

```diff
@@ -389,7 +389,7 @@ export function MachineLearningCommonUIProvider({
 
     async assertLastToastHeader(expectedHeader: string, timeout: number = 5000) {
       await retry.tryForTime(timeout, async () => {
-        const resultToast = await toasts.getToastElement(1);
+        const resultToast = await toasts.getToastElementByIndex(1);
         const titleElement = await testSubjects.findDescendant('euiToastHeader', resultToast);
         const title: string = await titleElement.getVisibleText();
```

Comment:
I see this pattern a few times - might be a candidate to add a new service method, e.g. something like `getToastTitleByIndex`.